datastores/action_list_datastore.ts

```diff
@@ -23,14 +26,11 @@ export const ActionListDatastore = DefineDatastore({
     status: {
       type: Schema.types.string,
     },
-    start_date: {
-      type: Schema.slack.types.date,
-    },
+    // start_date: {
```

Comment:
is there a reason this is commented out?

Reply:
I commented out start date cause I don't think that it would have been used in anything

Reply:
Removed commented code